ts/Series/FlowMap/FlowMapSeries.ts

```diff
@@ -0,0 +1,1019 @@
+         */
+        fillOpacity: 0.5,
+
+        /**
+         * An array specifying which option maps to which key in the data point
+         * array. This makes it convenient to work with unstructured data
+         * arrays from different sources.
+         *
+         * @default ['from', 'to', 'weight']
+         */
+        keys: ['from', 'to', 'weight'],
+
+        /**
+         * The [id](#series.id) of another series to link to. Additionally,
+         * the value can be ":previous" to link to the previous series. When
+         * two series are linked, only the first one appears in the legend.
+         * Toggling the visibility of this also toggles the linked series,
+         * which is necessary for operations such as zoom or updates on the
+         * flowmap series.
+         *
+         * @type      {string}
+         * @apioption plotOptions.flowmap.linkedTo
+         */
+
+        /**
+         * A `markerEnd` creates an arrow symbol
+         * indicating the direction of flow at the destination. Specifying a
+         * `markerEnd` here will create one for each link.
+         *
+         * @declare Highcharts.SeriesFlowMapSeriesOptionsObject
+         */
+        markerEnd: {
+            /**
+             * Enable or disable the `markerEnd`.
+             *
+             * @type   {boolean}
+             * @sample {highmaps} maps/series-flowmap/marker-end
+             *         Setting different markerType for markerEnd
+             */
+            enabled: true,
+            /**
+             * Height of the `markerEnd`. Can be a number in pixels
+             * or a percentage based on the weight of the link.
+             *
+             * @type  {number|string}
+             */
+            height: '40%',
+            /**
+             * Width of the `markerEnd`. Can be a number in pixels
+             * or a percentage based on the weight of the link.
+             *
+             * @type  {number|string}
+             */
+            width: '40%',
+            /**
+             * Change the shape of the `markerEnd`.
+             * Can be `arrow` or `mushroom`.
+             *
+             * @type {string}
+             */
+            markerType: 'arrow'
+        },
+
+        /**
+         * Maximum width of a link expressed in pixels. The weight of a link
+         * is mapped between `maxWidth` and `minWidth`.
+         *
+         * @type  {number}
+         */
+        maxWidth: 25,
+
+        /**
+         * Minimum width of a link expressed in pixels. The weight of a link
+         * is mapped between `maxWidth` and `minWidth`.
+         *
+         * @type  {number}
+         */
+        minWidth: 5,
+
+        /**
+         * The opacity of all the links.
+         *
+         * @apioption plotOptions.flowmap.opacity
+         */
+
+        /**
+         * The weight for all links with unspecified weights.
+         * The weight of a link determines its thickness compared to
+         * other links.
```

Comment:
Currently, everything is forced to be scaled by default.
Instead of adding a new option,  we could use `weight`, remove the default scaling (remove the default values for `minWidth` and `maxWidth`). If a check is added, the scaling will only be activated when they are actually set. In theory it should accomplish the above